src/explorer/Components/DistributionPublishers.cs/NumericSampleGenerator.cs

```diff
@@ -0,0 +1,40 @@
+namespace Explorer.Components
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+
+    using Accord.Statistics.Distributions.Univariate;
+    using Explorer.Common;
+    using Explorer.Metrics;
+
+    public class NumericSampleGenerator : EmpiricalDistributionPublisher
+    {
+        public const int DefaultSamplesToPublish = 20;
+        private readonly ExplorerContext ctx;
+
+        public NumericSampleGenerator(
+            ExplorerContext ctx,
+            ResultProvider<EmpiricalDistribution> distributionProvider)
+        : base(distributionProvider)
+        {
+            this.ctx = ctx;
+        }
+
+        public int SamplesToPublish { get; set; } = DefaultSamplesToPublish;
+
+        protected override IEnumerable<ExploreMetric> EnumerateMetrics(EmpiricalDistribution distribution)
+        {
+            yield return new UntypedMetric(
+                name: "sample_values",
+                metric: new
+                {
+                    Count = SamplesToPublish,
```

Comment:
The `Count` property is redundant here and interferes with the generation of sample data for multiple columns feature. The metric should contain just the list of values.

Reply:
You're right, I removed it. 
d6fb19a